dist/component/component-war/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -27,5 +27,7 @@
       <module name="org.jboss.resteasy.resteasy-jackson2-provider" services="import"/>
       <module name="org.hawkular.bus"/>
     </dependencies>
+    <module name="org.infinispan" export="true" />
+    <module name="org.infinispan.commons" export="true"/>
```

Comment:
The EARs already have an infinspan dep declared.  They don't have org.infinispan.commons declared, but iiuc this is not needed, because as I mentioned, this file should be getting ignored anyway.  Take it away and ensure that it still works as expected.